### PR TITLE
Update Version to 0.3.0

### PR DESCRIPTION
**Pull request by yunjayh**
This commit updates version to 0.3.0

ONE-vscode-DCO-1.0-Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

### CODE REVIEW

**Review by hyunsik-yoon — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
I will merge this PR after all bugfix PRs are merged :)

**Pull request merged**
